python/semantic_kernel/kernel.py

```diff
@@ -392,12 +431,24 @@ def on_function_invoked(self, function_view: FunctionView, context: KernelContex
             return args
         return None
 
-    def import_plugin(self, plugin_instance: Any, plugin_name: str = "") -> Dict[str, KernelFunctionBase]:
-        if plugin_name.strip() == "":
-            plugin_name = PluginCollection.GLOBAL_PLUGIN
-            logger.debug(f"Importing plugin {plugin_name} into the global namespace")
-        else:
-            logger.debug(f"Importing plugin {plugin_name}")
+    def import_plugin(self, plugin_instance: Any, plugin_name: str) -> KernelPluginBase:
+        """
+        Import a plugin into the kernel.
+
+        Args:
+            plugin_instance (Any): The plugin instance.
```

Comment:
I think it'd be nice if we defined a base class that plugins inherited, so we could be more explicit on the type. Right now, it can be any class that contains methods with the @kernal_function decorator.

Reply:
ok, then let's start with adding that sentence to the docstring!